errors.js

```diff
@@ -0,0 +1,10 @@
+exports.customErrors = (err, req, res, next) => {
+    if (err.status && err.msg) {
+      res.status(err.status).send({ msg: err.msg });
+    } else next(err);
+  };
+
+  exports.psqlErrors = (err, req, res, next) => {
+    if (err.code === "42703" || err.code === '22P02' || err.code === "23502") res.status(400).send({ msg: "Bad request" });
+    next(err);
+  };
```

Comment:
Each of these error codes do something different - 23502 is for insertion into columns with missing values, but you haven't done any insertions yet. Be careful not to add in bits of code that aren't doing anything just yet!